src/renderers/server-render.jsx

```diff
@@ -124,6 +118,15 @@ const getRouterRenderer = (
 	};
 };
 
+const makeRenderer$ = (config: Object) => makeRenderer(
+		config.routes,
+		config.reducer,
+		config.clientFilename,
+		config.assetPublicPath,
+		config.middleware,
+		config.baseUrl
+	);
```

Comment:
This unused constant should also be throwing lint errors - looks like we need to add `yarn lint` to the `test` script.